standup/status/models.py

```diff
@@ -12,7 +12,13 @@ class Project(models.Model):
     name = models.CharField(max_length=100, unique=True)
     slug = models.CharField(max_length=100, unique=True)
     color = models.CharField(max_length=6)
-    repo_url = models.URLField()
+    repo_url = models.URLField(max_length=100)
+
+    class Meta:
+        db_table = 'project'
```

Comment:
Mmm... Part of me likes this migration idea. But the other part of me is thinking about maintenance in the long term and we're going to end up with db tables that have names that have historical baggage.

What do you think? Is it "easy" to rename these tables later on? Am I overthinking this?